tests/console/validate_zfcp_multipath.pm

```diff
@@ -50,11 +56,23 @@ sub verify_multipath_block_devices {
     assert_equals 4, $small_disks, "40GB block device does not have 4 paths";
 }
 
+sub verify_multipath_block_devices_2_block {
+    my $mpath = script_output "multipath -l";
+    my @mpath = split /\n/, $mpath;
+    # check for 50GB disk (multiline match)
+    assert_matches qr/^3600507638081855cd800000000000076.+\n^size=50G/m, $mpath, "Cannot find 50GB disk on WWID *076";
+    # must see the LUN over 2 paths
+    my $disks = scalar grep { /\bsd(a|b)\b/ } @mpath;
+    assert_equals 2, $disks, "50GB block device does not have 2 paths";
+}
+
 sub run {
     select_console 'root-console';
     verify_host_bus_adapters_attached("0.0.fa00", "0.0.fc00");
-    verify_plain_scsi_block_devices;
-    verify_multipath_block_devices;
+    # verify_plain_scsi_block_devices;
```

Comment:
Do not need change here if we just update the old function.

Reply:
Done